### ISSUE TITLE
Brzydkie Handlinger og gester

### ISSUE THREAD

**Issue by pikrol** [pikrol/CDI-static#25]
![image](https://user-images.githubusercontent.com/44268776/108845877-3f4afa80-75de-11eb-828a-dd51d3db443b.png)

**Comment by karolina-m** [comment]
już chyba wiem! te pytania to kategoria "gestures", a kategoria gestures w WG w settings ma css_class = threeColumns i przez to tutaj one się wyświetlają w trzech kolumnach i zawijają się wiersze z odpowiedziami. Trzeba w settings dla wg_nor zmienić css_class dla gestures na puste (chyba na puste).

**Comment by pikrol** [comment]
Tak, można zrobić tak jak Pani pisze.  Mogę też ustawić większą szerokość tych kolumn i wtedy nie będą (chyba) się brzydko zawijać tylko jeśli nie zmieszczą się na ekranie trzy pełne to się zrobią dwie.

Może Pani to zobaczyć wchodząc na stronę z cdi, klikając prawym na pole z pytaniami i wybierając "zbadaj", następnie z okna, które powinno się wyświetlić po prawej (nie wiem czy tak jest tylko w Google Chrome czy w innych przeglądarkach też) trzeba wybrać <div class = "threeColumns"...  i w okienku na dole można zmienić column-width z 200px, które są obecnie np. na 300
